### PR TITLE
Make mention autocomplete case insensitive in new markdown editor

### PR DESCRIPTION
**Pull request by brechtvl**
This matches EasyMDE, and makes it easier to find the right user without having to remember the exact name.

### CODE REVIEW

**Review by silverwind — APPROVED** [review]
ah yes, an oversight from me.

**Comment by silverwind** [comment]
@brechtvl could you do the same for emoji a few lines above? so "1ST" should still capture "1st" emoji when doing ":" autocompletion.

File: web_src/js/features/comp/ComboMarkdownEditor.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ class ComboMarkdownEditor {
       } else if (key === '@') {
         const matches = [];
         for (const obj of window.config.tributeValues) {
-          if (obj.key.includes(text)) {
+          if (obj.key.toLowerCase().includes(text.toLowerCase())) {
```
Comment (wxiaoguang) [inline web_src/js/features/comp/ComboMarkdownEditor.js]: Could the `text.toLowerCase()` be a variable (out of the `for`) to slightly improve performance?

**Review by silverwind — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Comment by silverwind** [comment]
.

**Pull request merged**